### DISCUSSION TITLE
Relationship linking to non-ACM posts

### DISCUSSION THREAD

**Discussion by fluxism** [wpengine/atlas-content-modeler#573] [Ideas]
I'm testing the use of ACM for an events site that will sell event tickets. I've created the Events and Tickets models in ACM. I would like to link each Ticket to a corresponding Woocommerce product. Could the Relationship field be extended to link to non-ACM posts?

**Comment by mindctrl** [comment]
@fluxism thanks for the request! This is in our backlog, but I don't have an ETA on when we might add it. We'll update you here when we know more.
